src/usr/local/www/services_ntpd.php

```diff
@@ -210,6 +203,13 @@ function build_interface_list() {
 
 $section = new Form_Section('NTP Server Configuration');
 
+$section->addInput(new Form_Select(
+	'enable',
+	'Enable NTP Server',
+	$pconfig['enable'],
+	array('enabled' => 'Enabled', 'off' => 'Disabled')
+));
```

Comment:
A checkbox was not used on purpose, in order to get an _explicit_ value for disabled service -- which can be checked in the rest of the code, without people having to reconfigure anything, without any config upgrade and without any similar nonsense. IOW, so that it would keep working for everyone as it was for years. 

If you with to discuss whether a checkbox is better than dropdown or whether it's important to using false instead of off or disabled, have fun with this. 

P.S. I also am not responsible for the GET -> POST bugs that got spread across entire codebase and that cripple saving configuration.